config/batch/config.go

```diff
@@ -0,0 +1,39 @@
+package batch
+
+import (
+	"github.com/crossplane/upjet/pkg/config"
+	"github.com/upbound/provider-aws/config/common"
+)
+
+// Configure adds configurations for the batch group.
+func Configure(p *config.Provider) {
+	p.AddResourceConfigurator("aws_batch_compute_environment", func(r *config.Resource) {
+		r.References = config.References{
+			"compute_resources.instance_role": config.Reference{
+				TerraformName: "aws_iam_instance_profile",
+			},
+			"compute_resources.placement_group": config.Reference{
+				TerraformName: "aws_placement_group",
+			},
+			"service_role": config.Reference{
+				TerraformName: "aws_iam_role",
+			},
+			"compute_resources.subnets": config.Reference{
+				TerraformName: "aws_subnet",
+			},
+			"compute_resources.security_group_ids": config.Reference{
+				TerraformName: "aws_security_group",
+			},
+		}
+		r.UseAsync = true
+	})
+
+	p.AddResourceConfigurator("aws_batch_job_queue", func(r *config.Resource) {
+		r.References = config.References{
+			"compute_environment_order.compute_environment": config.Reference{
+				TerraformName: "aws_batch_compute_environment",
+				Extractor:     common.PathARNExtractor,
+			},
+		}
+	})
+}
```

Comment:
I tested these two resources in my local, and except for the automatically defined references, defining the above seems sufficient. Could you please remove the above and add only the following configurations?
```
func Configure(p *config.Provider) {
	p.AddResourceConfigurator("aws_batch_compute_environment", func(r *config.Resource) {
		r.References["compute_resources.subnets"] = config.Reference{
			TerraformName: "aws_subnet",
		}
		r.References["compute_resources.security_group_ids"] = config.Reference{
			TerraformName: "aws_security_group",
		}
	})
}
```